app/assets/javascripts/home.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
Let's remove this file

![](https://media.giphy.com/media/a9wtrSVIoefks/giphy.gif)

I'll list all the files you have to remove in the review description!